### PR TITLE
JA: translation /ja/install/docker.md

### PR DESCRIPTION
**Pull request by yukinagae**
This PR is just a split small version of this https://github.com/tensorflow/docs/pull/442 for reviewers to check the content of PR easily.

@sfujiwara You might have already reviewed `site/ja/installdocker.md`, and I did a small change on this version of the file to follow up the latest master branch. I only changed the TensorFlow version as below:

```
$ git diff 8cdbb909:site/ja/install/docker.md 4275213a:site/ja/install/docker.md
diff --git a/site/ja/install/docker.md b/site/ja/install/docker.md
index d6249dd9..c098b266 100644
--- a/site/ja/install/docker.md
+++ b/site/ja/install/docker.md
@@ -36,7 +36,7 @@ using the following format:
 | ---         | ---                                                                                               |
 | `latest`    | The latest release of TensorFlow CPU binary image. Default.                                       |
 | `nightly`   | Nightly builds of the TensorFlow image. (unstable)                                                |
-| *`version`* | Specify the *version* of the TensorFlow binary image, for example: *1.12.0*                       |
+| *`version`* | Specify the *version* of the TensorFlow binary image, for example: *1.13.1*                       |
 | `devel`     | Nightly builds of a TensorFlow `master` development environment. Includes TensorFlow source code. |
```

### CODE REVIEW

**Comment by masa-ita** [comment]
I think it's better to use "コンテナ"  instead of "container" in the first sentence.

How about "ほかの部分から分離した" instead of "他の部分から分離された" ?

Weak suggestion:
I prefer bold rather than italic for Japanese characters, because Japanese fonts don't have italic glyph.

Correction and suggestion :
"各ベーズtag" -> "各ベースタグ"

**Comment by masa-ita** [comment]
Thank you @yukinagae !
I reviewed changes you made. LGTM

**Comment by lamberta** [comment]
Thank you, @yukinagae 

We've recently arranged to translate the _toc.yaml and _index.yaml files using an internal process for Japanese and Korean. Since these files are updated frequently and unable to be previewed externally, it's too much of a burden on the community to expect them to maintain them.

We should start seeing those files in a few weeks :)
But, for now, please remove them from this pull request and I can merge and publish the docker.md
Thanks

File: site/ja/install/docker.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+コンテナ内の*ホスト*マシン上で開発されたTensorFlowプログラムを実行するには、
+ホストディレクトリをマウントしてコンテナの作業ディレクトリを変更します。
+(`-v hostDir:containerDir -w workDir`):
+
+<pre class="devsite-terminal devsite-click-to-copy prettyprint lang-bsh">
+docker run -it --rm -v $PWD:/tmp -w /tmp tensorflow/tensorflow python ./script.py
+</pre>
+
+コンテナ内で作成されたファイルがホストに公開されると、アクセス権限の問題が発生する可能性があります。
+通常はホストシステム上のファイルを編集するのがベストです。
```
[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/docker.md]: Fix: https://github.com/tensorflow/docs/pull/643/commits/24a61a35f7b8ee80a2e689ac66fae0496682bab0

File: site/ja/install/docker.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+イメージリリースは、次の形式で[タグ付け](https://hub.docker.com/r/tensorflow/tensorflow/tags/){:.external}されています:
+
+| タグ        | 説明                                                                                              |
+| ---         | ---                                                                                               |
+| `latest`    | 最新リリースのTensorFlowのCPUバイナリイメージ。デフォルト。                                       |
+| `nightly`   | TensorFlowのnightlyビルドのイメージ。(不安定)                                                     |
+| *`version`* | TensorFlowのバイナリイメージの*version*を指定する。例: *1.13.1*                                   |
+| `devel`     | TensorFlowのnightlyビルドの`master`開発環境。TensorFlowのソースコードを含む。                     |
+
+各ベーズ*tag*は、機能を追加または変更する亜種があります:
```
[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/docker.md]: Fix: https://github.com/tensorflow/docs/pull/643/commits/24a61a35f7b8ee80a2e689ac66fae0496682bab0

File: site/ja/install/docker.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
+# Docker
+
+[Docker](https://docs.docker.com/install/){:.external}は、*container*を使用して、
+TensorFlowのインストールをシステムの他の部分から分離された仮想環境を作成します。
```
[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/docker.md]: Thanks for your suggestion! 

@masa-ita gave me a comment on the same sentence you mentioned here and the below translation seems to be simple and straightforward to me. What do you think about it?

```
Dockerは、*コンテナ*を使用して、
TensorFlowのインストールをシステムのほかの部分から分離した仮想環境を作成します。
```

**Comment by yukinagae** [comment]
Thanks for reviewing the PR, @masa-ita 

You gave me a suggestion about using **bold** rather than *italic* for Japanese. To be honest, I don't have any idea on this point. I'd like to follow the community guideline for Japanese translation. If there is no such thing at the moment, we can discuss it via slack :) 

https://github.com/tensorflow/docs/pull/643#issuecomment-501692350

> Weak suggestion:
I prefer bold rather than italic for Japanese characters, because Japanese fonts don't have italic glyph.

File: site/ja/install/docker.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sfujiwara) [reply site/ja/install/docker.md]: OK, I thought that is a little bit strange as Japanese at first.
But, it was my misunderstanding.

First, I read the sentence as below:
`TensorFlow のインストール / を / システムのほかの部分から隔離された仮想環境 / を / 作成します`

But, you mean below (is that right?):
`TensorFlow のインストールをシステムのほかの部分から隔離された仮想環境 / を / 作成します`

I guess `xxx を xxx を作成します` might be a little bit confusing, but it's not wrong.
But, the above is only my opinion.

So, you can choose which you prefer and that is the right of translator!
I would agree to your decision:)

[the remark this answers is not in the document]
Comment (yukinagae) [reply site/ja/install/docker.md]: Yes, I meant the below sentence. I guess we can make it better, but I will keep my translation for now. Thanks for your suggestion!

**Comment by yukinagae** [comment]
Thank you @lamberta,  I removed _index.yaml and _toc.yaml from this PR for now: https://github.com/tensorflow/docs/pull/643/commits/7022de7a0d26d6fdf5529e13b589175d793b1edd

**Pull request merged**